packages/server/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "author": "Medplum <hello@medplum.com>",
   "scripts": {
-    "build": "npm run clean && tsc && tsc --project tsconfig.build.json",
+    "build": "npm run clean && tsc && tsc --project tsconfig.build.json && node -e \"require('node:fs').cpSync('src/img', 'dist/img', { recursive: true });\"",
```

Comment:
Added an `img` dir in `src` that we will always copy over. Using `node` to be platform agnostic

Reply:
Hmm... I don't think we want to copy this into `dist`

We should just add a `public` or `static` directory, and make sure that ends up in the Docker image.

I'd recommend handling this in `scripts/build-docker.sh`